src/views/CreateView.vue

```diff
@@ -224,6 +225,7 @@ export default {
   },
   data() {
     return {
+      allExercises: [],
```

Comment:
I'd call it just `exercises`, WDYT?

Reply:
Previously I had been afraid of a potential "collision" with `<transition name="exercises>` so I choose this name to avoid that but I changed for just `exercises` and everythin seems to work so you're probably right 🙃